test/shopping-list.test.js

```diff
@@ -0,0 +1,162 @@
+let chai = require('chai');
+let expect = chai.expect;
+let Item = require('../js/shopping-list-item.js');
+let List = require('../js/shopping-list.js');
+
+describe("Shopping list Item", function () {
+
+  describe("Item is a class", function () {
+
+    it("should be function", function () {
+      expect(Item).to.be.a("function");
+    })
+  })
+
+  describe("Item has a property", function () {
+    let steak = new Item('steak', 't-bone steak');
+    it("named 'name'", function () {
+      expect(steak).to.have.property('name');
+    })
+    it("named 'description'", function () {
+      expect(steak).to.have.property('description');
+    })
+    it('named "isDone"', function () {
+      expect(steak).to.have.property('isDone');
+    })
+  })
+
+  describe("has a method named check", function () {
+    let steak = new Item('steak', 't-bone steak');
+    it("should be a function", function () {
+      expect(steak.check).to.be.a("function");
+    })
+    it("should set items isDone property to true", function () {
+      steak.check();
+      expect(steak.isDone).to.be.true;
+    })
+  })
+
+  describe("has a method named uncheck", function () {
+    let steak = new Item('steak', 't-bone steak');
+    it("should be a function", function () {
+      expect(steak.uncheck).to.be.a("function");
+
+    })
+    it("should set items isDone property to false", function () {
+      steak.check();
+      steak.uncheck();
+      expect(steak.isDone).to.be.false;
+    })
+
+  })
+
+  describe("has a method named render", function () {
+    let steak = new Item('steak', 't-bone steak');
+    it("should be a function", function () {
+      expect(steak.render).to.be.a("function");
+    })
+
+    it("should return a string", function () {
+      expect(steak.render()).to.be.a("string");
+
+    })
+
+    it("should start with an <li tag", function () {
+      expect(steak.render().slice(0, 3)).to.equal('<li');
+
+    })
+    it("should end with an </li> tag", function () {
+      expect(steak.render().slice(-5)).to.equal('</li>');
+    })
+  })
```

Comment:
There should be an additional test to make sure that the string that's returned matches what you would expect it to look like and not just the beginning and the end parts.